docs/testplan/account-changes-testplan.md

```diff
@@ -17,7 +17,7 @@ This testplan is going to cover the new Free Trial period functionality. It will
 | Already existed user.                | 3. Existed user impact.                                            | For current customers, the free tier  will be sunsetting. We will email all users who are on the free trial four times: A 30-day notice, a 15-day notice, and a final notice on the final day.                                                                                                                                                                                                                                                                                                              |          |
 | Trial period configuration.          | 4. Trial period configuration - period changes.                    | Only new users should be affected.                                                                                                                                                                                                                                                                                                              |          |
 |                                      | 5. Trial period limits changes.                                    | Only new users should be affected.                                                                                                                                                                                                                                                                                      |          |
-| Migration.                           | 6. Migration command.                                              | Command should affect Free tier users. Migrated users should have 0 limits, and a “free trial expiration” event in the DB.                                                                                                                                                                                 |          |
+| Migration.                           | 6. Migration command.                                              | Command should affect Free tier users. users should have trial_expiration, which can be in the future. After that date passes the freeze chore should freeze user's account. NOTE: not to forget - the migration which will be run first: migrate free tier users who have legacy STORJ token payments to the paid tier - that way they are not affected by the main migration.                                                                                                                                                                                  |          |
```

Comment:
one more note: users migrated in the migration should not receive free trial notification emails, as they are being communicated with separately from the satellite